airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -14,6 +14,7 @@ data:
   registries:
     cloud:
       enabled: true
+      dockerImageTag: 1.2.0
```

Comment:
revert the dockerImageTag above as well if we know there is a bug in the latest release?

Reply:
ah sorry nvm i got wired crossed in that I thought we were also going to revert the entire PR back

Reply:
Dangit, didn't see this. I'll follow up with Support to see if we should do that.